drud/ddev-gitpod-launcher
# Add DDEV_REPO project to gitpod vscode Source Control panel

Testing instructions:
* Use this generated URL (which is using this PR's branch `shaal:add-project-in-source-control`)
* https://gitpod.io/#DDEV_REPO=https%3A%2F%2Fgithub.com%2Fshaal%2Ffldc22/https://github.com/shaal/ddev-gitpod-launcher/tree/add-project-in-source-control

* Open "Source Control" Panel in Gitpod, you should see 2 separate repos -
![image](https://user-images.githubusercontent.com/22901/144248083-69ab00fc-cd16-4807-900a-77eeb042e72f.png)


related to https://github.com/drud/ddev-gitpod-launcher/issues/2

<a href="https://gitpod.io/#https://github.com/drud/ddev-gitpod-launcher/pull/3"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
